source/common/json/config_schemas.cc

```diff
@@ -185,6 +185,7 @@ const std::string Json::Schema::LISTENER_SCHEMA(R"EOF(
        },
        "drain_type": {"type" : "string", "enum" : ["default", "modify_only"]},
        "ssl_context" : {"$ref" : "#/definitions/ssl_context"},
+       "transparent" : {"type": "boolean"},
```

Comment:
Is the v1 API absolutely required?

Reply:
We'll need it for Istio. Cf. https://github.com/istio/istio/pull/4654#pullrequestreview-108325780